src/data_processing/netflix_to_wikidata.py

```diff
@@ -11,7 +11,18 @@
 netflix_data_path = os.path.join(base_dir, 'netflix_movies.txt')
 output_csv_path = os.path.join(base_dir, 'netflix_to_wikidata.csv')
 
-netflix_data = pd.read_csv(netflix_data_path, delimiter=',', names=['NetflixId', 'Year', 'Title'],  encoding='ISO-8859-1', on_bad_lines='skip')
+netflix_data = []
+
+with open(netflix_data_path, mode='r', encoding='ISO-8859-1') as file:
+    reader = csv.reader(file, delimiter=',')
+    for row in reader:
+        if len(row) == 3: 
```

Comment:
As mentioned in Sofia's review, maybe we should just read the file line by line and do `split(',', 2)` in order to get movie titles with commas, as opposed to this strategy which just ignores them.